app/assets/javascripts/application.js

```diff
@@ -13,5 +13,34 @@
 //= require rails-ujs
 //= require turbolinks
 //= require jquery
-//= require bootstrap.bundle.min.js
 //= require_tree .
+
+document.addEventListener('DOMContentLoaded', function () {
+
+  // Get all "navbar-burger" elements
+  var $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);
```

Comment:
How much do we care about IE11? Because we should probably change this to 

```js
const navbarBurgers = document.querySelectorAll('.navbar-burgers');

navbarBurgers.forEach(e => e.addEventListener('click', function() {
 // Get the target from the "data-target" attribute
 const target = document.getElementById(e.dataset.target);

 // Toggle the class on both the "navbar-burger" and the "navbar-menu"
 e.classList.toggle('is-active');
 target.classList.toggle('is-active');
});
```

Also, there is no special convention that `$` holds in Javascript so avoid using that because frameworks usually use it to differentiate their services vs. the user written ones.

Reply:
Sounds good. Might as well care about ie 11